crates/eframe/src/web/web_painter_wgpu.rs

```diff
@@ -87,6 +87,10 @@ impl WebPainterWgpu {
     pub async fn new(canvas_id: &str, options: &WebOptions) -> Result<Self, String> {
         log::debug!("Creating wgpu painter");
 
+        if crate::web::web_location().hostname == "0.0.0.0" {
+            log::warn!("Due to a chromium bug, WebGPU doesn't work when the hostname is '0.0.0.0'");
```

Comment:
WebGPU is only available in secure contexts; that's anything HTTPS and a **few** whitelisted hostnames depending on browser, usually just `localhost`.

Reply:
Thanks for the clarification!

Reply:
ahh this makes a lot of sense! thanks!